android/quest/build.gradle.kts

```diff
@@ -171,11 +171,11 @@ android {
       manifestPlaceholders["appLabel"] = "MOH eCBIS"
     }
 
-    create("ecbis_preview") {
+    create("ecbis_training") {
```

Comment:
If we don't want to create a  new flavor we can just leave the old name but change the app label.

Unless we don't want to have a preview type flavor for ecbis moving forward

Reply:
We are completely renaming the flavor to training based on the clients team